### PR TITLE
Update args description of crop_to_bounding_box & pad_to_bounding_box

### LINKED ISSUE #61644: `tf.image.crop_to_bounding_box()` assumes `tf.int32` arguments, but not documented as such
### Issue type

Documentation Feature Request

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

binary

### TensorFlow version

2.13.0

### Custom code

No

### OS platform and distribution

Linux Ubuntu 22.04 (WSL 2)

### Mobile device

_No response_

### Python version

3.10

### Bazel version

_No response_

### GCC/compiler version

_No response_

### CUDA/cuDNN version

_No response_

### GPU model and memory

_No response_

### Current behavior?

`tf.image.crop_to_bounding_box()` implicitly assumes that the target width and height are `tf.int32`, but this is not documented anywhere. The cause for this is using `tf.shape()` which has the default `dtype` of `tf.int32`, in a stack operation:
https://github.com/tensorflow/tensorflow/blob/c9fafed9bc8cb0238a775fd4a0680e648c06b5b6/tensorflow/python/ops/image_ops_impl.py#L1250-L1254

### Standalone code to reproduce the issue

```python
import tensorflow as tf

image = tf.zeros([1000, 2000, 3], dtype=tf.uint8)
offset = tf.constant([0, 0], dtype=tf.int64)
size = tf.constant([900, 1500], dtype=tf.int64)
tf.image.crop_to_bounding_box(image, offset[0], offset[1], size[0], size[1])
```

### Relevant log output

```shell
Traceback (most recent call last):
  File "/.../test.py", line 6, in <module>
    tf.image.crop_to_bounding_box(image, offset[0], offset[1], size[0], size[1])
  File "/.../.venv/lib/python3.10/site-packages/tensorflow/python/util/traceback_utils.py", line 153, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/.../.venv/lib/python3.10/site-packages/tensorflow/python/framework/ops.py", line 6656, in raise_from_not_ok_status
    raise core._status_to_exception(e) from None  # pylint: disable=protected-access
tensorflow.python.framework.errors_impl.InvalidArgumentError: cannot compute Pack as input #1(zero-based) was expected to be a int32 tensor but is a int64 tensor [Op:Pack] name: stack
```

### PR DESCRIPTION
**Pull request by SuryanarayanaY**
Currently the documentation of the API `tf.image.crop_to_bounding_box` doesn't mentioning the acceptable dtypes of the arguments `offset_height, offset_width, target_height, target_width`. This API accepts only` tf.int32 Tensor` or a `integer`. For other dtypes its raising exception. Hence adding details of acceptable dtypes for these arguments for better clarity to the users.

Refer attached gist for [crop_to_bounding_box](https://colab.sandbox.google.com/gist/SuryanarayanaY/57ceb39be4649987d0cf3c9a3eb584f4/tf-image-crop_to_bounding_box.ipynb#scrollTo=xvm-YCU4X2Z0).

Also Fixes #61644 .

Similarly `tf.image.pad_to_bounding_box` documentation also have the same arguments `offset_height, offset_width, target_height, target_width`. But these arguments accept `int32` and `int64`Tensors but not others.Hence added details of acceptable dtypes for these arguments for better clarity to the users.

Refer attached gist for [pad_to_bounding_box](https://colab.sandbox.google.com/gist/SuryanarayanaY/c4bbec3161c740a73472c0361ba0959b/tf-image-pad_to_bounding_box.ipynb).

### CODE REVIEW

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -1084,10 +1084,14 @@ def pad_to_bounding_box_internal(image, offset_height, offset_width,
   Args:
     image: 4-D Tensor of shape `[batch, height, width, channels]` or 3-D Tensor
       of shape `[height, width, channels]`.
-    offset_height: Number of rows of zeros to add on top.
-    offset_width: Number of columns of zeros to add on the left.
-    target_height: Height of output image.
-    target_width: Width of output image.
+    offset_height: Number of rows of zeros to add on top.Must be 1-D `Tensor` 
+      of dtype int32 or int64.Can also a python integer.
```
Comment (cantonios) [inline tensorflow/python/ops/image_ops_impl.py:1088]: Missing space before "Can".

Comment (cantonios) [reply tensorflow/python/ops/image_ops_impl.py:1088]: Are you sure it's a 1-D tensor and not a 0-D tensor (i.e. a scalar)?  Here and throughout.

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -1084,10 +1084,14 @@ def pad_to_bounding_box_internal(image, offset_height, offset_width,
...
+    offset_width: Number of columns of zeros to add on the left.Must be
```
Comment (cantonios) [inline tensorflow/python/ops/image_ops_impl.py:1089]: Missing space before "Must".

**Review by cantonios — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 2 above)
Comment (SuryanarayanaY) [reply tensorflow/python/ops/image_ops_impl.py:1089]: Sure will add space at all required places.

**Review by cantonios — APPROVED** [review]

**Pull request merged**